website/pages/docs/reference/manifest.mdx

```diff
@@ -27,7 +27,6 @@ The package name is an identifier used to refer to the package.
 It is used when listed as a dependency in another package, and as the default name of targets.
 
 The name must use only ASCII alphanumeric characters or `_`, and cannot be empty.
-It cannot also start with underscore.
```

Comment:
```suggestion
It cannot also be equal to `_`.
```

Reply:
`_` is not a valid identifier in Cairo so that's already covered